federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -42,9 +42,9 @@ spec:
           mountPath: /srv/kubernetes/
           readOnly: true
       - name: etcd
-        image: quay.io/coreos/etcd:v2.3.3
+        image: gcr.io/google_containers/etcd:3.0.14-alpha.1
         command:
-          - /etcd
+          - /usr/local/bin/etcd
```

Comment:
@madhusudancs, I have made the federation-apiserver to use etcd2 as storage-backend.  going forward we need to migrate to etcd3 asap. also probably document the etcd migration procedure w.r.t federation.